Test/ParallelPrimitives/RadixSortKernels.h

```diff
@@ -90,7 +90,7 @@ extern "C" __global__ void CountKernel( int* gSrc, int* gDst, int gN, int gNItem
 
 	for( int i = threadIdx.x; i < BIN_SIZE; i += COUNT_WG_SIZE )
 	{
-		table[threadIdx.x] = 0;
+		table[i] = 0;
```

Comment:
This is not a problem since we fix the workgroup size of the count kernel to be the same as the bin size. But this change makes it more robust.